frontend/src/libs/components/icon/icon-name-to-plain-svg.ts

```diff
@@ -0,0 +1,21 @@
+import { ReactComponent as ArrowIcon } from '#assets/icons/arrow.svg';
```

Comment:
we should already have this mapper, you can remove this file and move your icons to existing one